inst/@sym/private/check_assumptions.m

```diff
@@ -0,0 +1,62 @@
+%% Copyright (C) 2016 Lagu
+%%
+%% This file is part of OctSymPy.
+%%
+%% OctSymPy is free software; you can redistribute it and/or modify
+%% it under the terms of the GNU General Public License as published
+%% by the Free Software Foundation; either version 3 of the License,
+%% or (at your option) any later version.
+%%
+%% This software is distributed in the hope that it will be useful,
+%% but WITHOUT ANY WARRANTY; without even the implied warranty
+%% of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
+%% the GNU General Public License for more details.
+%%
+%% You should have received a copy of the GNU General Public
+%% License along with this software; see the file COPYING.
+%% If not, see <http://www.gnu.org/licenses/>.
+
+%% -*- texinfo -*-
+%% @deftypefun check_assumptions (@var{x}, @var{dot})
+%% Check if the input have valid assumptions.
+%%
+%% Private helper function.
+%%
+%% @seealso{sym}
+%% @end deftypefun
+
+function check_assumptions(varargin)
+
```

Comment:
Helper to avoid write this in the sym file, and we can extend this for better functionality # # @